week-5/A-dom-manipulation/exercise.js

```diff
@@ -14,20 +22,45 @@ Task 2
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
 
+var alertButton = document.getElementById("alertBtn");
 
+alertButton.addEventListener("click", alertMessage);
 
+function alertMessage() {
+  alert("Thanks for visiting Bikes for Refugees!");
+}
 /*
 Task 3
 ======
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
 
+var addText = document.querySelector("#addTextBtn");
+
+addText.addEventListener("click", addMoreText);
 
+function addMoreText() {
+  var JumboElement = document.querySelector(".jumbotron");
+  var addParagraph = document.createElement("p");
 
+  addParagraph.innerHTML = `<p>Read more below.</p>`;
```

Comment:
This is correct, but what is the resulting DOM structure? I think it might be a little odd. How could we fix that?

Extra bonus question: if we didn't want to use `innerHTML` here, what could we use instead? Hint: text node.